appveyor.yml

```diff
@@ -48,6 +48,9 @@ before_test:
   - ps: $env:Path += ";C:\Program Files (x86)\Pandoc\"
   - pandoc -v
 
+after_build:
+  - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%]"

```

Comment:
@Yuri05 My hope is that this will be triggered and replace the tar ball already created...to see

Reply:
does not work quite yet. This is to early